sub-411400/sub-411400_ses-20181009T175037_behavior+image+ophys.nwb
Traceback (most recent call last):
  File "tools/populate_dandiset_asset_yamls.py", line 64, in main
    metadata = nwb2asset(
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 330, in nwb2asset
    asset = metadata2asset(metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 379, in metadata2asset
    return extract_model(models.BareAssetMeta, metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 234, in extract_model
    value = kwargs.get(field, extract_field(field, metadata))
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 303, in extract_field
    return FIELD_EXTRACTORS[field](metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 283, in extract_digest
    return {models.DigestType[metadata["digest_type"]]: metadata["digest"]}
  File "/home/dandi/miniconda3/lib/python3.8/enum.py", line 344, in __getitem__
    return cls._member_map_[name]
KeyError: 'sha256'

sub-411400/sub-411400_ses-20181011T174057_behavior+image+ophys.nwb
Traceback (most recent call last):
  File "tools/populate_dandiset_asset_yamls.py", line 64, in main
    metadata = nwb2asset(
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 330, in nwb2asset
    asset = metadata2asset(metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 379, in metadata2asset
    return extract_model(models.BareAssetMeta, metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 234, in extract_model
    value = kwargs.get(field, extract_field(field, metadata))
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 303, in extract_field
    return FIELD_EXTRACTORS[field](metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 283, in extract_digest
    return {models.DigestType[metadata["digest_type"]]: metadata["digest"]}
  File "/home/dandi/miniconda3/lib/python3.8/enum.py", line 344, in __getitem__
    return cls._member_map_[name]
KeyError: 'sha256'

sub-411400/sub-411400_ses-20181015T173410_behavior+image+ophys.nwb
Traceback (most recent call last):
  File "tools/populate_dandiset_asset_yamls.py", line 64, in main
    metadata = nwb2asset(
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 330, in nwb2asset
    asset = metadata2asset(metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 379, in metadata2asset
    return extract_model(models.BareAssetMeta, metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 234, in extract_model
    value = kwargs.get(field, extract_field(field, metadata))
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 303, in extract_field
    return FIELD_EXTRACTORS[field](metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 283, in extract_digest
    return {models.DigestType[metadata["digest_type"]]: metadata["digest"]}
  File "/home/dandi/miniconda3/lib/python3.8/enum.py", line 344, in __getitem__
    return cls._member_map_[name]
KeyError: 'sha256'

sub-411771/sub-411771_ses-20181003T174857_behavior+ophys.nwb
Traceback (most recent call last):
  File "tools/populate_dandiset_asset_yamls.py", line 64, in main
    metadata = nwb2asset(
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 330, in nwb2asset
    asset = metadata2asset(metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 379, in metadata2asset
    return extract_model(models.BareAssetMeta, metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 234, in extract_model
    value = kwargs.get(field, extract_field(field, metadata))
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 303, in extract_field
    return FIELD_EXTRACTORS[field](metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 283, in extract_digest
    return {models.DigestType[metadata["digest_type"]]: metadata["digest"]}
  File "/home/dandi/miniconda3/lib/python3.8/enum.py", line 344, in __getitem__
    return cls._member_map_[name]
KeyError: 'sha256'

sub-411771/sub-411771_ses-20181008T172946_behavior+ophys.nwb
Traceback (most recent call last):
  File "tools/populate_dandiset_asset_yamls.py", line 64, in main
    metadata = nwb2asset(
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 330, in nwb2asset
    asset = metadata2asset(metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 379, in metadata2asset
    return extract_model(models.BareAssetMeta, metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 234, in extract_model
    value = kwargs.get(field, extract_field(field, metadata))
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 303, in extract_field
    return FIELD_EXTRACTORS[field](metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 283, in extract_digest
    return {models.DigestType[metadata["digest_type"]]: metadata["digest"]}
  File "/home/dandi/miniconda3/lib/python3.8/enum.py", line 344, in __getitem__
    return cls._member_map_[name]
KeyError: 'sha256'

sub-411771/sub-411771_ses-20181003T174857_behavior+image+ophys.nwb
Traceback (most recent call last):
  File "tools/populate_dandiset_asset_yamls.py", line 64, in main
    metadata = nwb2asset(
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 330, in nwb2asset
    asset = metadata2asset(metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 379, in metadata2asset
    return extract_model(models.BareAssetMeta, metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 234, in extract_model
    value = kwargs.get(field, extract_field(field, metadata))
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 303, in extract_field
    return FIELD_EXTRACTORS[field](metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 283, in extract_digest
    return {models.DigestType[metadata["digest_type"]]: metadata["digest"]}
  File "/home/dandi/miniconda3/lib/python3.8/enum.py", line 344, in __getitem__
    return cls._member_map_[name]
KeyError: 'sha256'

sub-411771/sub-411771_ses-20181004T173816_behavior+image+ophys.nwb
Traceback (most recent call last):
  File "tools/populate_dandiset_asset_yamls.py", line 64, in main
    metadata = nwb2asset(
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 330, in nwb2asset
    asset = metadata2asset(metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 379, in metadata2asset
    return extract_model(models.BareAssetMeta, metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 234, in extract_model
    value = kwargs.get(field, extract_field(field, metadata))
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 303, in extract_field
    return FIELD_EXTRACTORS[field](metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 283, in extract_digest
    return {models.DigestType[metadata["digest_type"]]: metadata["digest"]}
  File "/home/dandi/miniconda3/lib/python3.8/enum.py", line 344, in __getitem__
    return cls._member_map_[name]
KeyError: 'sha256'

sub-411771/sub-411771_ses-20181008T172946_behavior+image+ophys.nwb
Traceback (most recent call last):
  File "tools/populate_dandiset_asset_yamls.py", line 64, in main
    metadata = nwb2asset(
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 330, in nwb2asset
    asset = metadata2asset(metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 379, in metadata2asset
    return extract_model(models.BareAssetMeta, metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 234, in extract_model
    value = kwargs.get(field, extract_field(field, metadata))
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 303, in extract_field
    return FIELD_EXTRACTORS[field](metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 283, in extract_digest
    return {models.DigestType[metadata["digest_type"]]: metadata["digest"]}
  File "/home/dandi/miniconda3/lib/python3.8/enum.py", line 344, in __getitem__
    return cls._member_map_[name]
KeyError: 'sha256'

sub-408021/sub-408021_ses-20180926T172917_behavior+image+ophys.nwb
Traceback (most recent call last):
  File "tools/populate_dandiset_asset_yamls.py", line 64, in main
    metadata = nwb2asset(
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 330, in nwb2asset
    asset = metadata2asset(metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 379, in metadata2asset
    return extract_model(models.BareAssetMeta, metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 234, in extract_model
    value = kwargs.get(field, extract_field(field, metadata))
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 303, in extract_field
    return FIELD_EXTRACTORS[field](metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 283, in extract_digest
    return {models.DigestType[metadata["digest_type"]]: metadata["digest"]}
  File "/home/dandi/miniconda3/lib/python3.8/enum.py", line 344, in __getitem__
    return cls._member_map_[name]
KeyError: 'sha256'

sub-408021/sub-408021_ses-20180927T182632_behavior+image+ophys.nwb
Traceback (most recent call last):
  File "tools/populate_dandiset_asset_yamls.py", line 64, in main
    metadata = nwb2asset(
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 330, in nwb2asset
    asset = metadata2asset(metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 379, in metadata2asset
    return extract_model(models.BareAssetMeta, metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 234, in extract_model
    value = kwargs.get(field, extract_field(field, metadata))
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 303, in extract_field
    return FIELD_EXTRACTORS[field](metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 283, in extract_digest
    return {models.DigestType[metadata["digest_type"]]: metadata["digest"]}
  File "/home/dandi/miniconda3/lib/python3.8/enum.py", line 344, in __getitem__
    return cls._member_map_[name]
KeyError: 'sha256'

sub-408021/sub-408021_ses-20181001T172833_behavior+image+ophys.nwb
Traceback (most recent call last):
  File "tools/populate_dandiset_asset_yamls.py", line 64, in main
    metadata = nwb2asset(
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 330, in nwb2asset
    asset = metadata2asset(metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 379, in metadata2asset
    return extract_model(models.BareAssetMeta, metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 234, in extract_model
    value = kwargs.get(field, extract_field(field, metadata))
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 303, in extract_field
    return FIELD_EXTRACTORS[field](metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 283, in extract_digest
    return {models.DigestType[metadata["digest_type"]]: metadata["digest"]}
  File "/home/dandi/miniconda3/lib/python3.8/enum.py", line 344, in __getitem__
    return cls._member_map_[name]
KeyError: 'sha256'

sub-411424/sub-411424_ses-20181009T152254_behavior+image+ophys.nwb
Traceback (most recent call last):
  File "tools/populate_dandiset_asset_yamls.py", line 64, in main
    metadata = nwb2asset(
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 330, in nwb2asset
    asset = metadata2asset(metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 379, in metadata2asset
    return extract_model(models.BareAssetMeta, metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 234, in extract_model
    value = kwargs.get(field, extract_field(field, metadata))
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 303, in extract_field
    return FIELD_EXTRACTORS[field](metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 283, in extract_digest
    return {models.DigestType[metadata["digest_type"]]: metadata["digest"]}
  File "/home/dandi/miniconda3/lib/python3.8/enum.py", line 344, in __getitem__
    return cls._member_map_[name]
KeyError: 'sha256'

sub-411424/sub-411424_ses-20181010T160230_behavior+image+ophys.nwb
Traceback (most recent call last):
  File "tools/populate_dandiset_asset_yamls.py", line 64, in main
    metadata = nwb2asset(
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 330, in nwb2asset
    asset = metadata2asset(metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 379, in metadata2asset
    return extract_model(models.BareAssetMeta, metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 234, in extract_model
    value = kwargs.get(field, extract_field(field, metadata))
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 303, in extract_field
    return FIELD_EXTRACTORS[field](metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 283, in extract_digest
    return {models.DigestType[metadata["digest_type"]]: metadata["digest"]}
  File "/home/dandi/miniconda3/lib/python3.8/enum.py", line 344, in __getitem__
    return cls._member_map_[name]
KeyError: 'sha256'

sub-411424/sub-411424_ses-20181011T155936_behavior+image+ophys.nwb
Traceback (most recent call last):
  File "tools/populate_dandiset_asset_yamls.py", line 64, in main
    metadata = nwb2asset(
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 330, in nwb2asset
    asset = metadata2asset(metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 379, in metadata2asset
    return extract_model(models.BareAssetMeta, metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 234, in extract_model
    value = kwargs.get(field, extract_field(field, metadata))
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 303, in extract_field
    return FIELD_EXTRACTORS[field](metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 283, in extract_digest
    return {models.DigestType[metadata["digest_type"]]: metadata["digest"]}
  File "/home/dandi/miniconda3/lib/python3.8/enum.py", line 344, in __getitem__
    return cls._member_map_[name]
KeyError: 'sha256'

sub-412933/sub-412933_ses-20181016T220226_behavior+image+ophys.nwb
Traceback (most recent call last):
  File "tools/populate_dandiset_asset_yamls.py", line 64, in main
    metadata = nwb2asset(
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 330, in nwb2asset
    asset = metadata2asset(metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 379, in metadata2asset
    return extract_model(models.BareAssetMeta, metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 234, in extract_model
    value = kwargs.get(field, extract_field(field, metadata))
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 303, in extract_field
    return FIELD_EXTRACTORS[field](metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 283, in extract_digest
    return {models.DigestType[metadata["digest_type"]]: metadata["digest"]}
  File "/home/dandi/miniconda3/lib/python3.8/enum.py", line 344, in __getitem__
    return cls._member_map_[name]
KeyError: 'sha256'

sub-412933/sub-412933_ses-20181017T221505_behavior+image+ophys.nwb
Traceback (most recent call last):
  File "tools/populate_dandiset_asset_yamls.py", line 64, in main
    metadata = nwb2asset(
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 330, in nwb2asset
    asset = metadata2asset(metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 379, in metadata2asset
    return extract_model(models.BareAssetMeta, metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 234, in extract_model
    value = kwargs.get(field, extract_field(field, metadata))
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 303, in extract_field
    return FIELD_EXTRACTORS[field](metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 283, in extract_digest
    return {models.DigestType[metadata["digest_type"]]: metadata["digest"]}
  File "/home/dandi/miniconda3/lib/python3.8/enum.py", line 344, in __getitem__
    return cls._member_map_[name]
KeyError: 'sha256'

sub-412933/sub-412933_ses-20181019T001829_behavior+image+ophys.nwb
Traceback (most recent call last):
  File "tools/populate_dandiset_asset_yamls.py", line 64, in main
    metadata = nwb2asset(
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 330, in nwb2asset
    asset = metadata2asset(metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 379, in metadata2asset
    return extract_model(models.BareAssetMeta, metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 234, in extract_model
    value = kwargs.get(field, extract_field(field, metadata))
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 303, in extract_field
    return FIELD_EXTRACTORS[field](metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 283, in extract_digest
    return {models.DigestType[metadata["digest_type"]]: metadata["digest"]}
  File "/home/dandi/miniconda3/lib/python3.8/enum.py", line 344, in __getitem__
    return cls._member_map_[name]
KeyError: 'sha256'

sub-412933/sub-412933_ses-20181022T225131_behavior+image+ophys.nwb
Traceback (most recent call last):
  File "tools/populate_dandiset_asset_yamls.py", line 64, in main
    metadata = nwb2asset(
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 330, in nwb2asset
    asset = metadata2asset(metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 379, in metadata2asset
    return extract_model(models.BareAssetMeta, metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 234, in extract_model
    value = kwargs.get(field, extract_field(field, metadata))
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 303, in extract_field
    return FIELD_EXTRACTORS[field](metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 283, in extract_digest
    return {models.DigestType[metadata["digest_type"]]: metadata["digest"]}
  File "/home/dandi/miniconda3/lib/python3.8/enum.py", line 344, in __getitem__
    return cls._member_map_[name]
KeyError: 'sha256'

sub-412933/sub-412933_ses-20181023T220932_behavior+image+ophys.nwb
Traceback (most recent call last):
  File "tools/populate_dandiset_asset_yamls.py", line 64, in main
    metadata = nwb2asset(
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 330, in nwb2asset
    asset = metadata2asset(metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 379, in metadata2asset
    return extract_model(models.BareAssetMeta, metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 234, in extract_model
    value = kwargs.get(field, extract_field(field, metadata))
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 303, in extract_field
    return FIELD_EXTRACTORS[field](metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 283, in extract_digest
    return {models.DigestType[metadata["digest_type"]]: metadata["digest"]}
  File "/home/dandi/miniconda3/lib/python3.8/enum.py", line 344, in __getitem__
    return cls._member_map_[name]
KeyError: 'sha256'

sub-412933/sub-412933_ses-20181025T222758_behavior+image+ophys.nwb
Traceback (most recent call last):
  File "tools/populate_dandiset_asset_yamls.py", line 64, in main
    metadata = nwb2asset(
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 330, in nwb2asset
    asset = metadata2asset(metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 379, in metadata2asset
    return extract_model(models.BareAssetMeta, metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 234, in extract_model
    value = kwargs.get(field, extract_field(field, metadata))
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 303, in extract_field
    return FIELD_EXTRACTORS[field](metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 283, in extract_digest
    return {models.DigestType[metadata["digest_type"]]: metadata["digest"]}
  File "/home/dandi/miniconda3/lib/python3.8/enum.py", line 344, in __getitem__
    return cls._member_map_[name]
KeyError: 'sha256'

sub-413663/sub-413663_ses-20181017T173352_behavior+image+ophys.nwb
Traceback (most recent call last):
  File "tools/populate_dandiset_asset_yamls.py", line 64, in main
    metadata = nwb2asset(
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 330, in nwb2asset
    asset = metadata2asset(metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 379, in metadata2asset
    return extract_model(models.BareAssetMeta, metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 234, in extract_model
    value = kwargs.get(field, extract_field(field, metadata))
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 303, in extract_field
    return FIELD_EXTRACTORS[field](metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 283, in extract_digest
    return {models.DigestType[metadata["digest_type"]]: metadata["digest"]}
  File "/home/dandi/miniconda3/lib/python3.8/enum.py", line 344, in __getitem__
    return cls._member_map_[name]
KeyError: 'sha256'

sub-413663/sub-413663_ses-20181018T173052_behavior+image+ophys.nwb
Traceback (most recent call last):
  File "tools/populate_dandiset_asset_yamls.py", line 64, in main
    metadata = nwb2asset(
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 330, in nwb2asset
    asset = metadata2asset(metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 379, in metadata2asset
    return extract_model(models.BareAssetMeta, metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 234, in extract_model
    value = kwargs.get(field, extract_field(field, metadata))
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 303, in extract_field
    return FIELD_EXTRACTORS[field](metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 283, in extract_digest
    return {models.DigestType[metadata["digest_type"]]: metadata["digest"]}
  File "/home/dandi/miniconda3/lib/python3.8/enum.py", line 344, in __getitem__
    return cls._member_map_[name]
KeyError: 'sha256'

sub-413663/sub-413663_ses-20181022T172824_behavior+image+ophys.nwb
Traceback (most recent call last):
  File "tools/populate_dandiset_asset_yamls.py", line 64, in main
    metadata = nwb2asset(
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 330, in nwb2asset
    asset = metadata2asset(metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 379, in metadata2asset
    return extract_model(models.BareAssetMeta, metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 234, in extract_model
    value = kwargs.get(field, extract_field(field, metadata))
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 303, in extract_field
    return FIELD_EXTRACTORS[field](metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 283, in extract_digest
    return {models.DigestType[metadata["digest_type"]]: metadata["digest"]}
  File "/home/dandi/miniconda3/lib/python3.8/enum.py", line 344, in __getitem__
    return cls._member_map_[name]
KeyError: 'sha256'

sub-418779/sub-418779_ses-20181113T015148_behavior+image+ophys.nwb
Traceback (most recent call last):
  File "tools/populate_dandiset_asset_yamls.py", line 64, in main
    metadata = nwb2asset(
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 330, in nwb2asset
    asset = metadata2asset(metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 379, in metadata2asset
    return extract_model(models.BareAssetMeta, metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 234, in extract_model
    value = kwargs.get(field, extract_field(field, metadata))
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 303, in extract_field
    return FIELD_EXTRACTORS[field](metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 283, in extract_digest
    return {models.DigestType[metadata["digest_type"]]: metadata["digest"]}
  File "/home/dandi/miniconda3/lib/python3.8/enum.py", line 344, in __getitem__
    return cls._member_map_[name]
KeyError: 'sha256'

sub-418779/sub-418779_ses-20181114T030141_behavior+image+ophys.nwb
Traceback (most recent call last):
  File "tools/populate_dandiset_asset_yamls.py", line 64, in main
    metadata = nwb2asset(
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 330, in nwb2asset
    asset = metadata2asset(metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 379, in metadata2asset
    return extract_model(models.BareAssetMeta, metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 234, in extract_model
    value = kwargs.get(field, extract_field(field, metadata))
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 303, in extract_field
    return FIELD_EXTRACTORS[field](metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 283, in extract_digest
    return {models.DigestType[metadata["digest_type"]]: metadata["digest"]}
  File "/home/dandi/miniconda3/lib/python3.8/enum.py", line 344, in __getitem__
    return cls._member_map_[name]
KeyError: 'sha256'

sub-418779/sub-418779_ses-20181115T010609_behavior+image+ophys.nwb
Traceback (most recent call last):
  File "tools/populate_dandiset_asset_yamls.py", line 64, in main
    metadata = nwb2asset(
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 330, in nwb2asset
    asset = metadata2asset(metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 379, in metadata2asset
    return extract_model(models.BareAssetMeta, metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 234, in extract_model
    value = kwargs.get(field, extract_field(field, metadata))
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 303, in extract_field
    return FIELD_EXTRACTORS[field](metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 283, in extract_digest
    return {models.DigestType[metadata["digest_type"]]: metadata["digest"]}
  File "/home/dandi/miniconda3/lib/python3.8/enum.py", line 344, in __getitem__
    return cls._member_map_[name]
KeyError: 'sha256'

sub-420011/sub-420011_ses-20181113T180512_behavior+image+ophys.nwb
Traceback (most recent call last):
  File "tools/populate_dandiset_asset_yamls.py", line 64, in main
    metadata = nwb2asset(
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 330, in nwb2asset
    asset = metadata2asset(metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 379, in metadata2asset
    return extract_model(models.BareAssetMeta, metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 234, in extract_model
    value = kwargs.get(field, extract_field(field, metadata))
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 303, in extract_field
    return FIELD_EXTRACTORS[field](metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 283, in extract_digest
    return {models.DigestType[metadata["digest_type"]]: metadata["digest"]}
  File "/home/dandi/miniconda3/lib/python3.8/enum.py", line 344, in __getitem__
    return cls._member_map_[name]
KeyError: 'sha256'

sub-420011/sub-420011_ses-20181114T180624_behavior+image+ophys.nwb
Traceback (most recent call last):
  File "tools/populate_dandiset_asset_yamls.py", line 64, in main
    metadata = nwb2asset(
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 330, in nwb2asset
    asset = metadata2asset(metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 379, in metadata2asset
    return extract_model(models.BareAssetMeta, metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 234, in extract_model
    value = kwargs.get(field, extract_field(field, metadata))
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 303, in extract_field
    return FIELD_EXTRACTORS[field](metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 283, in extract_digest
    return {models.DigestType[metadata["digest_type"]]: metadata["digest"]}
  File "/home/dandi/miniconda3/lib/python3.8/enum.py", line 344, in __getitem__
    return cls._member_map_[name]
KeyError: 'sha256'

sub-433414/sub-433414_ses-20190220T212644_behavior+image+ophys.nwb
Traceback (most recent call last):
  File "tools/populate_dandiset_asset_yamls.py", line 64, in main
    metadata = nwb2asset(
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 330, in nwb2asset
    asset = metadata2asset(metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 379, in metadata2asset
    return extract_model(models.BareAssetMeta, metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 234, in extract_model
    value = kwargs.get(field, extract_field(field, metadata))
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 303, in extract_field
    return FIELD_EXTRACTORS[field](metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 283, in extract_digest
    return {models.DigestType[metadata["digest_type"]]: metadata["digest"]}
  File "/home/dandi/miniconda3/lib/python3.8/enum.py", line 344, in __getitem__
    return cls._member_map_[name]
KeyError: 'sha256'

sub-433414/sub-433414_ses-20190221T202814_behavior+image+ophys.nwb
Traceback (most recent call last):
  File "tools/populate_dandiset_asset_yamls.py", line 64, in main
    metadata = nwb2asset(
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 330, in nwb2asset
    asset = metadata2asset(metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 379, in metadata2asset
    return extract_model(models.BareAssetMeta, metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 234, in extract_model
    value = kwargs.get(field, extract_field(field, metadata))
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 303, in extract_field
    return FIELD_EXTRACTORS[field](metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 283, in extract_digest
    return {models.DigestType[metadata["digest_type"]]: metadata["digest"]}
  File "/home/dandi/miniconda3/lib/python3.8/enum.py", line 344, in __getitem__
    return cls._member_map_[name]
KeyError: 'sha256'

sub-433414/sub-433414_ses-20190222T221848_behavior+image+ophys.nwb
Traceback (most recent call last):
  File "tools/populate_dandiset_asset_yamls.py", line 64, in main
    metadata = nwb2asset(
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 330, in nwb2asset
    asset = metadata2asset(metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 379, in metadata2asset
    return extract_model(models.BareAssetMeta, metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 234, in extract_model
    value = kwargs.get(field, extract_field(field, metadata))
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 303, in extract_field
    return FIELD_EXTRACTORS[field](metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 283, in extract_digest
    return {models.DigestType[metadata["digest_type"]]: metadata["digest"]}
  File "/home/dandi/miniconda3/lib/python3.8/enum.py", line 344, in __getitem__
    return cls._member_map_[name]
KeyError: 'sha256'

sub-433448/sub-433448_ses-20190220T230630_behavior+image+ophys.nwb
Traceback (most recent call last):
  File "tools/populate_dandiset_asset_yamls.py", line 64, in main
    metadata = nwb2asset(
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 330, in nwb2asset
    asset = metadata2asset(metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 379, in metadata2asset
    return extract_model(models.BareAssetMeta, metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 234, in extract_model
    value = kwargs.get(field, extract_field(field, metadata))
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 303, in extract_field
    return FIELD_EXTRACTORS[field](metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 283, in extract_digest
    return {models.DigestType[metadata["digest_type"]]: metadata["digest"]}
  File "/home/dandi/miniconda3/lib/python3.8/enum.py", line 344, in __getitem__
    return cls._member_map_[name]
KeyError: 'sha256'

sub-433448/sub-433448_ses-20190221T220634_behavior+image+ophys.nwb
Traceback (most recent call last):
  File "tools/populate_dandiset_asset_yamls.py", line 64, in main
    metadata = nwb2asset(
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 330, in nwb2asset
    asset = metadata2asset(metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 379, in metadata2asset
    return extract_model(models.BareAssetMeta, metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 234, in extract_model
    value = kwargs.get(field, extract_field(field, metadata))
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 303, in extract_field
    return FIELD_EXTRACTORS[field](metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 283, in extract_digest
    return {models.DigestType[metadata["digest_type"]]: metadata["digest"]}
  File "/home/dandi/miniconda3/lib/python3.8/enum.py", line 344, in __getitem__
    return cls._member_map_[name]
KeyError: 'sha256'

sub-433448/sub-433448_ses-20190225T214952_behavior+image+ophys.nwb
Traceback (most recent call last):
  File "tools/populate_dandiset_asset_yamls.py", line 64, in main
    metadata = nwb2asset(
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 330, in nwb2asset
    asset = metadata2asset(metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 379, in metadata2asset
    return extract_model(models.BareAssetMeta, metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 234, in extract_model
    value = kwargs.get(field, extract_field(field, metadata))
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 303, in extract_field
    return FIELD_EXTRACTORS[field](metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 283, in extract_digest
    return {models.DigestType[metadata["digest_type"]]: metadata["digest"]}
  File "/home/dandi/miniconda3/lib/python3.8/enum.py", line 344, in __getitem__
    return cls._member_map_[name]
KeyError: 'sha256'

sub-433448/sub-433448_ses-20190226T224301_behavior+image+ophys.nwb
Traceback (most recent call last):
  File "tools/populate_dandiset_asset_yamls.py", line 64, in main
    metadata = nwb2asset(
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 330, in nwb2asset
    asset = metadata2asset(metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 379, in metadata2asset
    return extract_model(models.BareAssetMeta, metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 234, in extract_model
    value = kwargs.get(field, extract_field(field, metadata))
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 303, in extract_field
    return FIELD_EXTRACTORS[field](metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 283, in extract_digest
    return {models.DigestType[metadata["digest_type"]]: metadata["digest"]}
  File "/home/dandi/miniconda3/lib/python3.8/enum.py", line 344, in __getitem__
    return cls._member_map_[name]
KeyError: 'sha256'

sub-433451/sub-433451_ses-20190215T165328_behavior+image+ophys.nwb
Traceback (most recent call last):
  File "tools/populate_dandiset_asset_yamls.py", line 64, in main
    metadata = nwb2asset(
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 330, in nwb2asset
    asset = metadata2asset(metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 379, in metadata2asset
    return extract_model(models.BareAssetMeta, metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 234, in extract_model
    value = kwargs.get(field, extract_field(field, metadata))
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 303, in extract_field
    return FIELD_EXTRACTORS[field](metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 283, in extract_digest
    return {models.DigestType[metadata["digest_type"]]: metadata["digest"]}
  File "/home/dandi/miniconda3/lib/python3.8/enum.py", line 344, in __getitem__
    return cls._member_map_[name]
KeyError: 'sha256'

sub-433451/sub-433451_ses-20190218T162355_behavior+image+ophys.nwb
Traceback (most recent call last):
  File "tools/populate_dandiset_asset_yamls.py", line 64, in main
    metadata = nwb2asset(
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 330, in nwb2asset
    asset = metadata2asset(metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 379, in metadata2asset
    return extract_model(models.BareAssetMeta, metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 234, in extract_model
    value = kwargs.get(field, extract_field(field, metadata))
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 303, in extract_field
    return FIELD_EXTRACTORS[field](metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 283, in extract_digest
    return {models.DigestType[metadata["digest_type"]]: metadata["digest"]}
  File "/home/dandi/miniconda3/lib/python3.8/enum.py", line 344, in __getitem__
    return cls._member_map_[name]
KeyError: 'sha256'

sub-433451/sub-433451_ses-20190219T162113_behavior+image+ophys.nwb
Traceback (most recent call last):
  File "tools/populate_dandiset_asset_yamls.py", line 64, in main
    metadata = nwb2asset(
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 330, in nwb2asset
    asset = metadata2asset(metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 379, in metadata2asset
    return extract_model(models.BareAssetMeta, metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 234, in extract_model
    value = kwargs.get(field, extract_field(field, metadata))
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 303, in extract_field
    return FIELD_EXTRACTORS[field](metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 283, in extract_digest
    return {models.DigestType[metadata["digest_type"]]: metadata["digest"]}
  File "/home/dandi/miniconda3/lib/python3.8/enum.py", line 344, in __getitem__
    return cls._member_map_[name]
KeyError: 'sha256'

sub-433458/sub-433458_ses-20190221T162838_behavior+image+ophys.nwb
Traceback (most recent call last):
  File "tools/populate_dandiset_asset_yamls.py", line 64, in main
    metadata = nwb2asset(
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 330, in nwb2asset
    asset = metadata2asset(metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 379, in metadata2asset
    return extract_model(models.BareAssetMeta, metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 234, in extract_model
    value = kwargs.get(field, extract_field(field, metadata))
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 303, in extract_field
    return FIELD_EXTRACTORS[field](metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 283, in extract_digest
    return {models.DigestType[metadata["digest_type"]]: metadata["digest"]}
  File "/home/dandi/miniconda3/lib/python3.8/enum.py", line 344, in __getitem__
    return cls._member_map_[name]
KeyError: 'sha256'

sub-433458/sub-433458_ses-20190222T161948_behavior+image+ophys.nwb
Traceback (most recent call last):
  File "tools/populate_dandiset_asset_yamls.py", line 64, in main
    metadata = nwb2asset(
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 330, in nwb2asset
    asset = metadata2asset(metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 379, in metadata2asset
    return extract_model(models.BareAssetMeta, metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 234, in extract_model
    value = kwargs.get(field, extract_field(field, metadata))
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 303, in extract_field
    return FIELD_EXTRACTORS[field](metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 283, in extract_digest
    return {models.DigestType[metadata["digest_type"]]: metadata["digest"]}
  File "/home/dandi/miniconda3/lib/python3.8/enum.py", line 344, in __getitem__
    return cls._member_map_[name]
KeyError: 'sha256'

sub-433458/sub-433458_ses-20190227T162140_behavior+image+ophys.nwb
Traceback (most recent call last):
  File "tools/populate_dandiset_asset_yamls.py", line 64, in main
    metadata = nwb2asset(
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 330, in nwb2asset
    asset = metadata2asset(metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 379, in metadata2asset
    return extract_model(models.BareAssetMeta, metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 234, in extract_model
    value = kwargs.get(field, extract_field(field, metadata))
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 303, in extract_field
    return FIELD_EXTRACTORS[field](metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 283, in extract_digest
    return {models.DigestType[metadata["digest_type"]]: metadata["digest"]}
  File "/home/dandi/miniconda3/lib/python3.8/enum.py", line 344, in __getitem__
    return cls._member_map_[name]
KeyError: 'sha256'

sub-440889/sub-440889_ses-20190306T163903_behavior+image+ophys.nwb
Traceback (most recent call last):
  File "tools/populate_dandiset_asset_yamls.py", line 64, in main
    metadata = nwb2asset(
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 330, in nwb2asset
    asset = metadata2asset(metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 379, in metadata2asset
    return extract_model(models.BareAssetMeta, metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 234, in extract_model
    value = kwargs.get(field, extract_field(field, metadata))
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 303, in extract_field
    return FIELD_EXTRACTORS[field](metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 283, in extract_digest
    return {models.DigestType[metadata["digest_type"]]: metadata["digest"]}
  File "/home/dandi/miniconda3/lib/python3.8/enum.py", line 344, in __getitem__
    return cls._member_map_[name]
KeyError: 'sha256'

sub-440889/sub-440889_ses-20190314T152429_behavior+image+ophys.nwb
Traceback (most recent call last):
  File "tools/populate_dandiset_asset_yamls.py", line 64, in main
    metadata = nwb2asset(
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 330, in nwb2asset
    asset = metadata2asset(metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 379, in metadata2asset
    return extract_model(models.BareAssetMeta, metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 234, in extract_model
    value = kwargs.get(field, extract_field(field, metadata))
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 303, in extract_field
    return FIELD_EXTRACTORS[field](metadata)
  File "/home/dandi/cronlib/dandi-api-datasets/.cronlib/venv/lib/python3.8/site-packages/dandi/metadata.py", line 283, in extract_digest
    return {models.DigestType[metadata["digest_type"]]: metadata["digest"]}
  File "/home/dandi/miniconda3/lib/python3.8/enum.py", line 344, in __getitem__
    return cls._member_map_[name]
KeyError: 'sha256'

